code/obj/submachine/cooking.dm

```diff
@@ -146,6 +161,56 @@ TYPEINFO(/obj/submachine/chef_sink)
 		..()
 
 
+/datum/action/bar/private/handwashing/critterwashing
```

Comment:
What behaviour does this inherit from the handwashing action? Is this inheritnce necessary?

Reply:
it originally had more in common, but grew in scope when critter ai got involved. on it